src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private static final int MAX = 100;

```

Comment:
It is a good practice to declare constants first. Please, place constant declaration before regular class fields.
Also, think about more descriptive name. Maybe, `MAX_BALL_NUMBER` would more descriptive?